### PR TITLE
removed 'cache' option as it was not implemented

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

 - Docs Pull Request

##### ANSIBLE VERSION

```
>=2.5
```

### CODE REVIEW

**Comment by mattclay** [comment]
CI failure in unit tests: https://app.shippable.com/github/ansible/ansible/runs/70733/4/tests

Integration test failures appear unrelated.

File: changelogs/fragments/script_cache_no.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+    - removed docs for config for cache in script plugin as it was not implemented
```
Comment (gundalow) [inline changelogs/fragments/script_cache_no.yml:2]: suggestion: script inventory plugin

**Comment by mattclay** [comment]
CI failure in unit tests: https://app.shippable.com/github/ansible/ansible/runs/76205/3/tests

**Comment by mattclay** [comment]
CI failure in unit tests: https://app.shippable.com/github/ansible/ansible/runs/82808/3/tests

**Pull request closed without merging**